src/main/java/com/eprosima/fastcdr/idl/templates/TypesSource.stg

```diff
@@ -919,18 +807,18 @@ default:
 b = true;
 switch(__d)
 {
-    $totallabels:{case $it$:}; separator="\n"$
+    $totallabels:{ label | case $label$:}; separator="\n"$
     b = false;
     break;
     default:
     break;
```

Comment:
Indentation

```suggestion
    b = true;
    switch (__d)
    {
        $totallabels:{ label | case $label$:}; separator="\n"$
            b = false;
            break;
        default:
            break;
```

These changes also apply below to the `$else$` case.

Reply:
Done in 3d798c2